_maps/RandomZLevels/SnowCabin.dmm

```diff
@@ -4017,7 +4013,7 @@
 	name = "Who"
 	},
 /obj/item/clothing/head/helmet/knight/yellow{
-	armor = list("melee" = 11, "bullet" = 2, "laser" = 1, "energy" = 1, "bomb" = 5, "bio" = 2, "fire" = 0, "acid" = 10);
+	armor = list("melee"=11,"bullet"=2,"laser"=1,"energy"=1,"bomb"=5,"bio"=2,"fire"=0,"acid"=10);
```

Comment:
This is all just from pressing Ctrl+S on StrongDMM

Reply:
why in gods name

Reply:
why are we doing armor overrides in a fucking dmm

Reply:
Assuming this is an insta-fail with the new armor changes, it's because ruins might not be run in CI

Reply:
> Assuming this is an insta-fail with the new armor changes, it's because ruins might not be run in CI

absolutely, this will cause nonstop runtimes the second anything tries to grab armor

Reply:
If it's when they try to *grab* armor then, assuming we *are* loading ruins in CI (outside of compile maps), it means that we need to be checking this on Initialize (most certainly inside a `TEST_ONLY_ASSERT`)

Reply:
> If it's when they try to _grab_ armor then, assuming we _are_ loading ruins in CI (outside of compile maps), it means that we need to be checking this on Initialize (most certainly inside a `TEST_ONLY_ASSERT` or `PERFORM_ALL_TESTS`)

I can make that a check yeah